### DISCUSSION TITLE
Not working on my Linux VPS, works fine on localhost windows.

### DISCUSSION THREAD

**Discussion by RemcoRol** [osiset/laravel-shopify#717] [General]
Hi there,

Thanks for this great library! I got a strange event occurring on my Linux VPS where I want to host my Shopify app.

I didn't know where to really post it because I am not sure whether this is a bug or not!

When I go to my VPS's url I see the missingDomain error but that's fine because it doens't contain the 'shop' parameter but when I add the shop parameter it's giving me back a 404? Anyone has an idea how this is possible?

http://h2854111.stratoserver.net/ < works fine

http://h2854111.stratoserver.net?shop=yourshop.shopify.com returns the screenshot below..

![image](https://user-images.githubusercontent.com/7826753/109973114-4bc70580-7cf8-11eb-952e-662af23f201b.png)

**Comment by gnikyt** [comment]
You dont have rewrite rules setup with Apache

**Reply by RemcoRol** [reply]
I think that's already good as it shows the first page fine and I am hosting multiple websites this way?

Rewrite module is already on:

sudo a2enmod rewrite
Module rewrite already enabled

my configuration file;

<VirtualHost *:80>
    DocumentRoot /var/www/html/SureShotBFGConnector/public/

    <Directory "/var/www/html/SureShotBFGConnector/public/">
            Options FollowSymLinks
            ReWriteEngine On
    </Directory>

    ErrorLog ${APACHE_LOG_DIR}/error.log
    CustomLog ${APACHE_LOG_DIR}/access.log combined
</VirtualHost>

**Reply by gnikyt** [reply]
Sorry I don't know your specific issue but this is a config issue with your rewrite or mapping it to the PHP handler, as you're getting a 404 when using a fake URL.
